### PR TITLE
LambdaTest integration tile

### PR DESCRIPTION
**Pull request by prateeksainilambdatest**
### What does this PR do?

LambdaTest integration tile initial release.

### CODE REVIEW

**Comment by alai97** [comment]
Created editorial review card DOCS-2825.

**Comment by hithwen** [comment]
Hi, the tests on this PR are broken because of an error on our setup. Once this is merged, if you update your branch the build should be able to run fine https://github.com/DataDog/integrations-extras/pull/1088#pullrequestreview-815828267

File: lambdatest/README.md
Comment (sarah-witt) [inline lambdatest/README.md:21]:
```suggestion
All configuration happens on Lambdatest Dashboard. Find details on how to set it up in the [LambdaTest-Datadog integration Documentation][1]
```

**Review by sarah-witt — COMMENTED** [review]
Hi @prateeksainilambdatest, thanks for the contribution. Does this integration submit any service checks, metrics, or logs?

File: lambdatest/README.md
Comment (prateeksainilambdatest) [reply lambdatest/README.md]: This integration does not submit any service checks, metrics or logs.

**Review by sarah-witt — APPROVED** [review]
Looks good to me! I will wait for the documentation team to review.

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:9]:
```suggestion
With the LambdaTest integration, log bugs while performing cross-browser testing of your websites (and web apps) using LambdaTest platform. LambdaTest will automatically include details from the testing environment, such as browser version, OS, resolution, comments, and screenshots in Datadog.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:12]:
```suggestion

- Live-interactive testing across 2000+ browsers and real machines hosted on cloud-based infrastructure.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:13]:
```suggestion
- An online automation testing grid supports Selenium and Cypress tests with any CI/CD pipeline to help QA teams validate and ship quality builds faster.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:14]:
```suggestion
- A next-gen developer-friendly browser that helps to build well-performing and responsive websites with speed.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:21]:
```suggestion
All configuration happens on the Lambdatest Dashboard. See the [LambdaTest-Datadog integration][1] setup documentation.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:24]:
```suggestion

Here’s how you can track incidents in Datadog with LambdaTest:
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:26]: A link here to the login site would be helpful

Comment (sarina-dd) [inline lambdatest/README.md:26]:
```suggestion
1. Log in to your LambdaTest account on the LambdaTest website.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:29]:
```suggestion
4. To integrate Datadog, click on install next to Push to Datadog.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:30]:
```suggestion
5. Enter your [Datadog API and application key][2].
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:42]:
```suggestion
[1]: https://www.lambdatest.com/support/docs/datadog-integration/
[2]: https://docs.datadoghq.com/account_management/api-app-keys/
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:31]:
```suggestion
6. A confirmation email is sent. Confirm the integration in the email.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:32]:
```suggestion
7. Once Datadog is integrated with your LambdaTest account, start logging bugs and performing cross-browser testing.
```

**Comment by ChristineTChen** [comment]
Hi @prateeksainilambdatest 

Friendly bump in case you didn't see the Documentation team's review.

Also there is a conflict in `.github/CODEOWNERS`.

**Comment by prateeksainilambdatest** [comment]
> Hi @prateeksainilambdatest
> 
> Friendly bump in case you didn't see the Documentation team's review.
> 
> Also there is a conflict in `.github/CODEOWNERS`.

Thanks for the reminder. I'll check

**Comment by prateeksainilambdatest** [comment]
Made all the necessary changes. Please review.

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:1]:
```suggestion
# LambdaTest integration
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:9]:
```suggestion
With the LambdaTest integration, log bugs while performing cross-browser testing of your websites (and web apps) using LambdaTest platform. LambdaTest automatically includes details from the testing environment, such as browser version, OS, resolution, comments, and screenshots in Datadog.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:22]:
```suggestion
All configuration happens on the LambdaTest Dashboard. See the [LambdaTest-Datadog integration][1] setup documentation.
```

File: lambdatest/README.md
Comment (sarina-dd) [inline lambdatest/README.md:25]:
```suggestion

Here’s how you can track incidents in Datadog with LambdaTest:
```

**Review by sarina-dd — APPROVED** [review]
Small additional edits, otherwise LGTM 👍

**Comment by ChristineTChen** [comment]
@prateeksainilambdatest, please see the final documentation reviews. I can merge once completed. Thanks!

**Comment by prateeksainilambdatest** [comment]
All requested changes updated. Please merge.

File: lambdatest/README.md
Comment (prateeksainilambdatest) [reply lambdatest/README.md]: Updated

File: lambdatest/README.md
Comment (prateeksainilambdatest) [reply lambdatest/README.md]: Updated

File: lambdatest/README.md
Comment (prateeksainilambdatest) [reply lambdatest/README.md]: Updated

File: lambdatest/README.md
Comment (prateeksainilambdatest) [reply lambdatest/README.md:26]: Updated

**Review by ChristineTChen — APPROVED** [review]

**Pull request merged**
